fmriprep/workflows/bold/apply.py

```diff
@@ -25,6 +26,108 @@
     from niworkflows.utils.spaces import SpatialReferences
 
 
+def init_bold_volumetric_resample_wf(
+    *,
+    metadata: dict,
+    fieldmap_id: ty.Optional[str] = None,
```

Comment:
since we're already using the cleaner method (imo)
```suggestion
    fieldmap_id: str | None = None,
```